src/main/java/com/amazon/opendistroforelasticsearch/security/support/ConfigHelper.java

```diff
@@ -71,13 +73,16 @@ public static void uploadFile(Client tc, String filepath, String index, CType cT
         try (Reader reader = createFileOrStringReader(cType, configVersion, filepath, populateEmptyIfFileMissing)) {
 
             final String res = tc
-                    .index(new IndexRequest(index).type(configVersion==1?"security":"_doc").id(cType.toLCString()).setRefreshPolicy(RefreshPolicy.IMMEDIATE)
+                    .index(new IndexRequest(index).type(configVersion==1?"security":"_doc").id(cType.toLCString()).opType(DocWriteRequest.OpType.CREATE).setRefreshPolicy(RefreshPolicy.IMMEDIATE)
                             .source(cType.toLCString(), readXContent(reader, XContentType.YAML))).actionGet().getId();
 
             if (!cType.toLCString().equals(res)) {
                 throw new Exception("   FAIL: Configuration for '" + cType.toLCString()
                         + "' failed for unknown reasons. Pls. consult logfile of elasticsearch");
             }
+            LOGGER.info("Doc '{}' with version {} is updated.", cType, configVersion);
+        } catch (VersionConflictEngineException versionConflictEngineException) {
+            LOGGER.warn("Doc with id {} in index {} already exists. Doing nothing.", cType.toLCString(), index);
```

Comment:
IMO, the warning level is not appropriate as it is not a misconfiguration or something went wrong. On a multi-node cluster, only one node will update the index, all other nodes are supposed to fail and would log the warning message.

Reply:
What do you think is the appropriate level ? 
IMO the node tried to do something and failure is expected so put as warning. These logs are on the node level.

Reply:
It depends on who is the target of the message. If it is intended for ES admin, it should be INFO (but admin should be able to do something, like delete index or upload it manually) or DEBUG, if it is intended for plugin developers.